### ISSUE TITLE
Improve layout of sub-sections in sidebar navigation

### ISSUE THREAD

**Issue by Idadelveloper** [unicef/inventory-hugo-theme#37]
# Summary

The sidebar of the cohort pages is getting too long with the addition of new start-ups. It's even worse on mobile devices where the user will have to scroll endlessly before reaching the main content. (https://unicef.github.io/inventory/cohorts/blockchain/)
It will be better if just the 5 main categories of the cohorts (_Blockchain, Drones, AR+VR, Data Science & AI, and Others_) are the only links on the sidebar.

**Screenshots**:
![unicef-sidebar](https://user-images.githubusercontent.com/56026636/144093969-a5387df2-9619-44ee-ba03-96b7d18c24fd.png)

**Comment by jwflory** [comment]
Moved from `unicef/inventory#101`. I also added this issue to the project board for the next sprint.

A couple of thoughts for this task:

1. Instead of removing the sidebar, we should either use a more responsive layout, or make them collapsible once they get past a certain count.
1. This change should be made for the sidebar for all pages instead of only the cohorts section.

Instead of removing the sub-sections, I think we should keep them there for navigation. But when the list gets long, we should definitely make some kind of improvement there so it doesn't fill the whole page. This is an issue that also happens when browsing other sections of the site, like the meta category:

https://unicef.github.io/inventory/meta/

I see two possible approaches for improving this with Bootstrap components:

1. [**Collapse component**](https://getbootstrap.com/docs/4.1/components/collapse/): Make sections collapsible once they reach a certain length, e.g. >5 sub-sections.
1. [**Responsive navbar component**](https://getbootstrap.com/docs/4.1/components/navbar/#responsive-behaviors): Convert the sidebar to a Bootstrap navbar, and make it responsive by default with a toggler.
